### ISSUE TITLE
Relative paths in Loading data not wrorking

### ISSUE THREAD

**Issue by pmooij** [wmeints/mlnet-tutorial#14]
Hi Willem,
thanks for the AI Night workshop yesterday! 
for getting the solution up-and-running in VS2019 I had to fix the relative paths in Trainer.Program, I changed them to absolute paths as workaround. Would be nice if they work out-of-the-box.

**Comment by wmeints** [comment]
Thanks for reporting the issue. Can you tell me a little bit more about the project directory layout?
The tutorial assumes a project layout that looks like this:

```
D:.
├───Data
├───Model
├───Trainer
└───Website
```

**Comment by pmooij** [comment]
it's exactly like that!

I had the issue with VS2019, because there the working folder is .\Trainer\bin\Debug\netcoreapp2.2

I see you [work around that handy in VSCode](https://github.com/wmeints/mlnet-tutorial/blob/master/src/completed/.vscode/launch.json) by specifying **"cwd": "${workspaceFolder}/Trainer"**
By using VSCode with that launch.json - although targeted at netcoreapp2.2 instead of netcoreapp3.0 - it does work flawless!

Is there a similar approach for setting the working folder for VS2019 as well?

so basically the issue is more about this being mentioned from your tutorial, neither when copying the _starter_ folder nor building it from scratch, like I did. 

But your _completed_ folder helped out though 👍

**Comment by wmeints** [comment]
I can definitely add a launch.json and tasks.json to the starter project. 
I'll also take a look in VS2019 to see if I can set up some default properties for that use-case as well.
